<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgStreamOptions</class>
 <widget class="QDialog" name="dlgStreamOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="lblDataType">
        <property name="text">
         <string>Data Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbDataType">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>2 channel unsigned 8 bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 channel signed 8 bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 channel unsigned 16 bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 channel signed 16 bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 channel float 32bit</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgtCF" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lblCF">
        <property name="text">
         <string>Center Frequency (Hz)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leCF">
        <property name="text">
         <string>100000000</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>126</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgtSampleRate" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblSampleRate">
        <property name="text">
         <string>Sample Rate (SPS)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leSampleRate">
        <property name="text">
         <string>5000000</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>147</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgtBandwidth" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblBandwidth">
        <property name="text">
         <string>Bandwidth (Hz)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leBandwidth">
        <property name="text">
         <string>5000000</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>156</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cmdOK">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>125</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
